lisa/tools/blkid.py

```diff
@@ -33,6 +35,9 @@ class Blkid(Tool):
     # /dev/sda1: LABEL="Temporary Storage" UUID="9ED4084BD408285B" TYPE="ntfs" PARTUUID="03e90eae-01" # noqa: E501
     _get_partition_name = re.compile(r"\s*(?P<name>\S+):.*")
 
+    # /dev/sda1: LABEL="Temporary Storage" UUID="9ED4084BD408285B" TYPE="ntfs" PARTUUID="03e90eae-01" # noqa: E501
```

Comment:
The original logic is low efficient, run multiple regexp on the same string. Please merge them, and call `find_group_in_lines`.
Use https://regex101.com/ to debug regexp.